engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetCullFace.java

```diff
@@ -55,45 +40,30 @@ public StateChange getDefaultInstance() {
     }
 
     @Override
-    public RenderPipelineTask generateTask() {
-        if (task == null) {
-            task = new SetCullFaceTask(mode);
+    protected RenderPipelineTask getDisablingTask() {
+        if (disablingTask == null) {
+            disablingTask = new DisableCapabilityTask(CAPABILITY);
         }
 
-        return task;
+        return disablingTask;
     }
 
     @Override
-    public boolean isEqualTo(StateChange stateChange) {
-        if (stateChange instanceof SetCullFace) {
-            return mode == ((SetCullFace) stateChange).getMode();
+    protected RenderPipelineTask getEnablingTask() {
+        if (enablingTask == null) {
+            enablingTask = new EnableCapabilityTask(CAPABILITY);
         }
-        return false;
+
+        return enablingTask;
     }
 
     @Override
     public boolean isTheDefaultInstance() {
-        return defaultInstance == this;
-    }
-
-    public static String getModeName(int mode) {
-        String modeName = "N/A";
-        switch (mode) {
-            case GL11.GL_BACK:
-                modeName = "GL_BACK";
-                break;
-            case GL11.GL_FRONT:
-                modeName = "GL_FRONT";
-                break;
-            case GL11.GL_FRONT_AND_BACK:
-                modeName = "GL_FRONT_AND_BACK";
-                break;
-        }
-        return modeName;
+        return this == defaultInstance;
```

Comment:
Not possible, because of Java. Declaring a static variable in super will not generate copies of the same static variable inside every subclass.